GOVERNANCE.rst

```diff
@@ -0,0 +1,183 @@
+
+The technical committee (TC) members are core developers who have additional rights and responsibilities to avoid deadlocks and to ensure the smooth running of the project. TC members are expected to participate in strategic planning, and approve changes to the governance model.
+
+The purpose of the TC is to ensure a smooth progress from the big-picture perspective. Changes that impact the full project require a synthetic analysis and a consensus that is both explicit and informed. In cases that the core developer community (which includes the TC members) fails to reach a consensus in the required time frame, the TC is the entity to resolve the
+issue.
+
+Membership of the TC is by nomination by a core developer. A nomination will result in discussion which cannot take more than a week and then a vote by the core developers which will stay open for a week. TC membership votes are subject to a two-third majority of all cast votes as well as a simple majority approval of all the current TC members.
+
+TC members who do not actively engage with the TC duties are expected to resign.
+
+The initial members of the TC are:
+
+* Markus Löning - @mloning
+* Franz Király - @fkiraly
+* Anthony Bagnall - @TonyBagnall
+
+Voting: lazy consensus with veto right
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When no consensus can be found, any core developer can call for a vote at any point during the discussion. The vote will conclude one week from the call for the vote.
+
+The vote is a binary vote: for (+1) or against (-1) accepting the proposed changes. All votes take place on the issue or pull request.
+
+If no option can gather two thirds of the votes cast, the decision is escalated to the TC, which in turn will use consensus seeking with the fallback option of a simple majority vote if no consensus can be found within a month. Any TC decision must be backed by an enhancement proposal.
+
+Decisions (in addition to adding core developers and TC membership as above) are made according to the following rules:
+
+* **Additions**, such as new algorithms: Requires +1 by one core developer, no -1 by a core developer (lazy consensus), happens on the issue or PR page.
+* **Minor documentation changes**, such as typo fixes, or addition/correction of a sentence: Requires +1 by one core developer, no -1 by a core developer (lazy consensus), happens on the pull request page. Core developers are expected to give “reasonable time” to others to give their opinion on the pull request if they’re not confident others would agree.
+* **Code changes and major documentation changes** require +1 by one core developers, no -1 by a core developer or code maintainer (lazy consensus), happens on the pull-request page.
+* **Changes to the API design and changes to dependencies or supported versions** happen via an enhancement proposal and follows the decision-making process outlined above.
+* **Changes to the governance model** use the same decision process outlined above.
+
```

Comment:
the rules are different per category. So, when the category is in doubt, who decides?

Reply:
the formulation on the condition for changes to the governance model is not clear. "same as above", but above there are multiple processes that are non-identical.

Reply:
When in doubt, the TC decides.

Reply:
makes sense, but perhaps should be explicit?